### ISSUE TITLE
Organization Wide Validation Profile

### ISSUE THREAD

**Issue by shahamit** [Apicurio/apicurio-studio#1609]
Is it possible to set the validation profile for all api specs design on the studio instance? 

We would like to have a common validation profile that is used by every api designer designing specs for his modules/features. As I understand any api designer or collaborate can change the validation profile and hence it seems the profile configuration is at the user level. 

Is there some config to set a single validation profile for any new or existing specs?

Thanks.

**Comment by EricWittmann** [comment]
You are right that the current feature is set at the user level.  We don't have a organization-wide setting for this, but it's a good idea.  I've made this an enhancement request.

**Comment by shahamit** [comment]
Ok great. Is there a way to export and import a validation profile right now? I can at least workaround it through this way until this feature comes out.

Thanks

**Comment by EricWittmann** [comment]
We don't have that, but that's another great idea.  @Apicurio/developers or @SafarMirek anyone with some bandwidth that could implement export/import of validation profiles in Studio?  It should be a relatively trivial feature to add I think!
